ccd-definition/CaseField.json

```diff
@@ -702,5 +702,38 @@
     "FieldType": "Collection",
     "FieldTypeParameter": "Recitals",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "actionCaseManagementOrder",
+    "Label": " ",
+    "FieldType": "ActionCaseManagementOrder",
```

Comment:
The name of the type and field sounds like method name since it starts with verb. Think of types as they were classes. You would not name class to read like a method, would you?